src/Event/FootprintListener.php

```diff
@@ -19,7 +19,7 @@ class FootprintListener implements EventListenerInterface
     protected $_defaultConfig = [
         'events' => [
             'Model.beforeFind' => -100,
-            'Model.beforeSave' => -100,
+            'Model.beforeRules' => -100,
```

Comment:
That redundancy is necessary to ensure the foot print in available in "beforeSave" if someone calls `save()` with `'checkRules' => false`.